hail/python/hail/linalg/blockmatrix.py

```diff
@@ -1112,7 +1115,8 @@ def to_numpy(self):
 
         Notes
         -----
-        The number of entries must be less than :math:`2^{31}`.
+        The number of entries must be less than :math:`2^{31}`. If the number of entries is
+        larger than this, consider using :meth:`.export_blocks` and :meth:`.rectangles_to_numpy`.
```

Comment:
let's delete the note entirely and modify this method to use the slower but safe approach if `n_rows * n_cols > 1 << 31`

Reply:
PS, have we actually tried this on a humongous matrix? it would be nice to try it on a 70k x 70k matrix once to verify there's no issues.

Reply:
Works! Might not be fast but takes no longer than a trip to lunch